docs/getting-started.md

```diff
@@ -154,7 +154,8 @@ generate_study_population: Cleaning up container and volume
 The final line tells you a file of (random) patient data has been created at
 `output/input.csv`, and that it would normally be considered highly sensitive
 data. What you see here reproduces exactly what would happen on a real, secure
-server, but with *dummy data* which contains no real patient information.
+server, but with *dummy data* which contains no real patient information. 
+Remember this is a blank study, read on below to generate *dummy data* values.
```

Comment:
Good idea, although re-reading it in context I'm not sure we've really signposted that this is a blank study - so instead of "Remember...", might be better as "Because we haven't modified the template blank study yet, this file is empty - we'll generate some *dummy data* values in the next section" - wdyt?

Reply:
heh, crossed in the post @sebbacon

Reply:
I think that is a much better wording suggestion @madwort  thanks!

 @sebbacon I know I'm not target audience but I was confused enough to stop and wonder where the dummy data